vuepress/docs/next/docs/getting-started/entando-cli.md

```diff
@@ -7,35 +7,43 @@ sidebarDepth: 1
 The Entando Command Line Interface, **ent**, provides a set of commands that accelerate common tasks such as installing a new instance of Entando, building projects, creating and deploying bundles, and composing Entando Applications. This document covers the install, command list, and a few operations to start using the ent tool.
 
 For specific topics, follow these links:
-1. [ent bundle: Bundle Management](ent-bundle.md)
-2. [ent api: API Management](ent-api.md)
-3. [ent svc: Services Management](ent-svc.md)
-4. [ent profile: Profile Management](ent-profile.md)
-5. [ent diag: Diagnostics and Troubleshooting](ent-diag.md)
+1. [Bundle Management](ent-bundle.md): `ent bundle`
+2. [API Management](ent-api.md): `ent bundle api`
+3. [Services Management](ent-svc.md): `ent bundle svc`
+4. [Profile Management](ent-profile.md): `ent profile`
+5. [Diagnostics and Troubleshooting](ent-diag.md): `ent diag`
 
 ### Prerequisites
 
-The basic requirements for the CLI vary depending on the type of developer tasks, as shown in the table below. The ent tool can install secondary dependencies using the `ent check-env` command as described [here](#check-the-environment).
+The basic requirements for the CLI vary with use case, as shown in the table below. 
 
 | Category | Prerequisite
 | :- | :-
-|Basic Development| Git or [Git for Windows](https://gitforwindows.org)
+| Basic Development| Git or [Git for Windows](https://gitforwindows.org)
 | Install Entando in a local VM | [Multipass](https://multipass.run/#install)
 | Build, Package and Publish Entando Bundles | Docker and Docker Compose
 | | A Docker Hub account for Docker images
 | Deploy an Entando Bundle | A Kubernetes cluster with admin access or a shared remote cluster
 
->The automatic option in [Getting Started](../getting-started/) will install the CLI along with a quickstart Entando Application.
 
 ## Install the CLI
-Install the current official release of the CLI with the following command:
+Install the current official release of the CLI:
 ``` bash
 curl -L https://get.entando.org/cli | bash
 ```
 
+>The automatic option in [Getting Started](../getting-started/) will install the CLI along with a quickstart Entando Application.
+
+### Check the Environment
```

Comment:
think it makes more organizational sense for the page as a whole to install the cli, see the list of commands and then show the notable commands below it

Reply:
disagree... before you can work with ent / execute commands you need to make sure the install is good and dependencies are satisfied. like with the M1 chip, you can go ahead and install ent and if you just start trying commands nothing will work because there are missing dependencies

Reply:
actually in the last version "ent check-env develop" was part of the prerequisites section, before the commands... but that meant the check was listed twice on the page